packages/asgardex-bitcoin/__tests__/client.test.ts

```diff
@@ -192,4 +192,45 @@ describe('BitcoinClient Test', () => {
       btcClient.transfer({ asset: AssetBTC, recipient: invalidAddress, amount, feeRate: 1 }),
     ).rejects.toThrow(expectedError)
   })
+
+
+  it('should get address transactions', async () => {
+    btcClient.setNetwork('testnet')
+
+    const txPages = await btcClient.getTransactions({ address:addyThree, limit: 4  })
+    expect(txPages.total).toEqual(1)
+    expect(txPages.txs[0].asset).toEqual(AssetBTC)
+    expect(txPages.txs[0].date).toEqual(new Date("2020-10-07T16:28:33.000Z"))
```

Comment:
Test of the date failed running the tests on my machine:
```bash
  ● BitcoinClient Test › should get address transactions

    expect(received).toEqual(expected) // deep equality

    Expected: 2020-10-07T16:28:33.000Z
    Received: 2020-10-07T23:28:33.000Z
```

Reply:
I wasn't able to replicate this error. Could you please try this one again.
![image](https://user-images.githubusercontent.com/9801383/95933888-8b08be80-0e0a-11eb-840e-7a2600d4613a.png)

Reply:
Upon further investigation this seems to be a UTC vs Timezone date parse issue. I've fixed the issue on bitcoin client side. Please let me know if you have this issue again.